packages/zendesk-adapter/src/filters/brand_logo.ts

```diff
@@ -83,7 +91,9 @@ const getBrandLogo = async ({ client, brand }: {
   )
   const pathName = pathNaclCase(name)
 
-  const content = await getLogoContent(client, brand)
+  // eslint-disable-next-line camelcase
+  const { id, file_name } = brand.value.logo
```

Comment:
you can remove the linter warning with
```suggestion
  // eslint-disable-next-line camelcase
  const { id, file_name: fileName } = brand.value.logo
```